src/DataTypes/DataTypeUUID.h

```diff
@@ -18,6 +18,8 @@ class DataTypeUUID : public IDataType
     static constexpr auto type_id = TypeIndex::UUID;
 
     const char * getFamilyName() const override { return "UUID"; }
+    const char * getSQLCompatibleName() const override { return "CHAR"; }
```

Comment:
CHAR in MySQL is of fixed size. Specifying CHAR without a size can lead to something strange.

Reply:
Although it might work...